mesheryctl/internal/cli/root/registry/publish.go

```diff
@@ -146,6 +142,9 @@ mesheryctl registry publish website $CRED 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwiz
 		case "remote-provider":
 			err = remoteProviderSystem()
 		case "website":
+			if outputFormat != "md" && outputFormat != "mdx" && outputFormat != "js" {
```

Comment:
.svg is to be written out here as well?

Reply:
Why is this condition needed? What's bug / issue is it fixing?

Reply:
This condition makes sure that user proveded right output format, otherwise we show error